dds.py

```diff
@@ -1,8 +1,22 @@
 from gql import Client
 from gql.transport.requests import RequestsHTTPTransport
 
-DDS_ENDPOINT = "https://dds.example.com/graphql"
+# Configure these to match the settings on your Dash Deployment Server:
 
-transport = RequestsHTTPTransport(url=DDS_ENDPOINT, use_json=True)
+# Domain name, from the settings in the Plotly On-Premise Server Manager (ask
+# your local administrator if you don't know this).
+DDS_DOMAIN_NAME = "dash.example.com"
+# Username and API key; you can obtain an API key from the Dash App Manager.
+AUTH = ("username", "api_key")
+# SSL/TLS certificate verification can be disabled here FOR TEST/DEMO SYSTEMS
+# ONLY.
+VERIFY = True
+
+# It's safe to leave this option unchanged:
+DDS_ENDPOINT = "https://{}/Manager/graphql".format(DDS_DOMAIN_NAME)
+
+transport = RequestsHTTPTransport(
+    url=DDS_ENDPOINT, auth=AUTH, use_json=True, verify=VERIFY
+)
 
 client = Client(retries=3, transport=transport)
```

Comment:
@scjody I know we included this in the initial example, but not sure if users want 3 retries by default 🙂Just a thought!

Reply:
OK, I'll remove it.